### PR TITLE
13370 Fix Incorporate a numbered ben company btn and wrong color for login btn text

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#13370

*Description of changes:*
- Deleted unused function which did the same thing as startNumberedCompany
- changed account id fetching so it works for staff and users
- Make text white for Home log in btn
- App version = 1.1.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/home/HomeView.vue [hunk 1]
```diff
@@ -565,8 +565,4 @@ export default class HomeView extends Vue {
     background-color: transparent;
   }
 
-  .hero-banner-login-btn {
```
Comment (lambert-alex) [inline auth-web/src/views/auth/home/HomeView.vue:568]: This css is no longer needed it was only changing the one buttons text to the wrong color.

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 2]
```diff
@@ -507,12 +507,6 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
   close () {
     this.$refs.errorDialog.close()
   }
-
-  private goToManageBusinesses (isNumberedCompanyRequest: boolean = true): void {
```
Comment (lambert-alex) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:511]: This method does the exact same thing as startNumberedCompany except it wouldn't refresh the data table.

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 3]
```diff
@@ -336,7 +336,7 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
 
   // create a numbered company
   protected async startNumberedCompany () {
-    await this.createNumberedBusiness(this.currentAccountSettings.id)
+    await this.createNumberedBusiness(this.currentOrganization.id)
```
Comment (lambert-alex) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:339]: currentOrganization.id works for both staff and user to return a valid id unlike currentAccountSettings

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by seeker25 — APPROVED** [review]

**Comment by seeker25** [comment]
Looks good on DEV, pushing to test

**Pull request merged**